actions/image_partition_action.go

```diff
@@ -250,6 +250,20 @@ func (i ImagePartitionAction) getPartitionDevice(number int, context debos.Debos
 	}
 }
 
+func (i *ImagePartitionAction) triggerDeviceNodes(context *debos.DebosContext) error {
+	// find the raw /dev entry
+	device, _ := filepath.EvalSymlinks(context.Image)
+
+	// trigger udev rules
+	err := debos.Command{}.Run("udevadm", "udevadm", "trigger", device)
```

Comment:
This will only trigger e.g. `/dev/vda`; You probably want to use `--parent-match`  to trigger the raw disk and all its children; Also probably wise to add `--settle` so we know for sure udev had processed all events before exiting this function